### ISSUE TITLE
Non-fatal exception thrown within HtmlAggregateStoryReporter

### ISSUE THREAD

**Issue by nbarrett** [serenity-bdd/serenity-core#968]
Currently using serenity 1.5.7. This maybe a repercussion of me using a HtmlAggregateStoryReporter directly (due to my running in a non-maven/gradle environment), but when the following code is run:

```
        HtmlAggregateStoryReporter htmlAggregateStoryReporter = new HtmlAggregateStoryReporter(SystemEnvironmentVariables.createEnvironmentVariables().getProperty(ThucydidesSystemProperty.SERENITY_PROJECT_NAME));
        htmlAggregateStoryReporter.setGenerateTestOutcomeReports();
        htmlAggregateStoryReporter.setOutputDirectory(rootDirectory);
         htmlAggregateStoryReporter.generateReportsForTestResultsFrom(rootDirectory);
```
The following WARN is logged:

```
13-Oct-2017 14:25:08.568 WARN  [main] n.t.c.r.h.HtmlAggregateStoryReporter - Failed to generate report for net.thucydides.core.reports.html.TagTypeReportingTask@6f20adb6 - java.util.concurrent.ExecutionException: java.lang.RuntimeException: Failed to merge template: Parsing error in template
net.thucydides.core.reports.html.Merger$MergeBuilder.to(Merger.java:39)
java.util.concurrent.ExecutionException: java.lang.RuntimeException: Failed to merge template: Parsing error in template
	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.util.concurrent.FutureTask.get(FutureTask.java:206)
	at net.thucydides.core.reports.html.Reporter.generateReports(Reporter.java:59)
	at net.thucydides.core.reports.html.Reporter.generateReportsFor(Reporter.java:35)
	at net.thucydides.core.reports.html.HtmlAggregateStoryReporter.generateReportsForTestResultsIn(HtmlAggregateStoryReporter.java:197)
	at net.thucydides.core.reports.html.HtmlAggregateStoryReporter.generateReportsForTestResultsFrom(HtmlAggregateStoryReporter.java:134)
	at uk.gov.dwp.universe.util.Reports.aggregate(Reports.java:37)
	at SerenityBatRunner.tearDown(SerenityBatRunner.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at cucumber.api.junit.Cucumber.run(Cucumber.java:100)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:51)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: java.lang.RuntimeException: Failed to merge template: Parsing error in template
	at net.thucydides.core.reports.html.Merger$MergeBuilder.to(Merger.java:39)
	at net.thucydides.core.reports.html.BaseReportingTask.generateReportPage(BaseReportingTask.java:46)
	at net.thucydides.core.reports.html.TagTypeReportingTask.generateTagTypeReport(TagTypeReportingTask.java:60)
	at net.thucydides.core.reports.html.TagTypeReportingTask.generateTagTypeReportsFor(TagTypeReportingTask.java:44)
	at net.thucydides.core.reports.html.TagTypeReportingTask.generateReports(TagTypeReportingTask.java:32)
	at net.thucydides.core.reports.html.ReportExecutor.call(ReportExecutor.java:32)
	at net.thucydides.core.reports.html.ReportExecutor.call(ReportExecutor.java:11)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: net.thucydides.core.reports.templates.TemplateMergeException: Parsing error in template
	at net.thucydides.core.reports.templates.FreemarkerReportTemplate.<init>(FreemarkerReportTemplate.java:27)
	at net.thucydides.core.reports.templates.FreeMarkerTemplateManager.getTemplateFrom(FreeMarkerTemplateManager.java:32)
	at net.thucydides.core.reports.html.Merger$MergeBuilder.to(Merger.java:36)
	... 10 common frames omitted
Caused by: freemarker.core.ParseException: Syntax error in template "freemarker/results-by-tagtype.ftl" in line 55, column 83:
Encountered "/", but was expecting one of:
    <STRING_LITERAL>
    <RAW_STRING>
    "false"
    "true"
    <INTEGER>
    <DECIMAL>
    "."
    "+"
    "-"
    "!"
    "["
    "("
    "{"
    <ID>
	at freemarker.core.FMParser.generateParseException(FMParser.java:5251)
	at freemarker.core.FMParser.jj_consume_token(FMParser.java:5122)
	at freemarker.core.FMParser.UnaryExpression(FMParser.java:450)
	at freemarker.core.FMParser.MultiplicativeExpression(FMParser.java:559)
	at freemarker.core.FMParser.AdditiveExpression(FMParser.java:512)
	at freemarker.core.FMParser.RangeExpression(FMParser.java:682)
	at freemarker.core.FMParser.RelationalExpression(FMParser.java:662)
	at freemarker.core.FMParser.EqualityExpression(FMParser.java:600)
	at freemarker.core.FMParser.AndExpression(FMParser.java:745)
	at freemarker.core.FMParser.OrExpression(FMParser.java:768)
	at freemarker.core.FMParser.Expression(FMParser.java:348)
	at freemarker.core.FMParser.StringOutput(FMParser.java:1240)
	at freemarker.core.FMParser.MixedContent(FMParser.java:3036)
	at freemarker.core.FMParser.OptionalBlock(FMParser.java:3253)
	at freemarker.core.FMParser.Root(FMParser.java:3432)
	at freemarker.template.Template.<init>(Template.java:208)
	at freemarker.cache.TemplateCache.loadTemplate(TemplateCache.java:495)
	at freemarker.cache.TemplateCache.getTemplate(TemplateCache.java:409)
	at freemarker.cache.TemplateCache.getTemplate(TemplateCache.java:261)
	at freemarker.template.Configuration.getTemplate(Configuration.java:1786)
	at freemarker.template.Configuration.getTemplate(Configuration.java:1646)
	at net.thucydides.core.reports.templates.FreemarkerReportTemplate.<init>(FreemarkerReportTemplate.java:25)
	... 12 common frames omitted
13-Oct-2017 14:25:08.570 WARN  [main] n.t.c.r.h.HtmlAggregateStoryReporter - SOME REPORT PAGES COULD NOT BE GENERATED
 * Failed to generate report for net.thucydides.core.reports.html.TagTypeReportingTask@6f20adb6 - java.util.concurrent.ExecutionException: java.lang.RuntimeException: Failed to merge template: Parsing error in template
net.thucydides.core.reports.html.Merger$MergeBuilder.to(Merger.java:39)


13-Oct-2017 14:25:08.571 INFO  [main] n.t.c.r.h.HtmlAggregateStoryReporter - Test results for 1 tests generated in 2450 ms

```

I still seem to get a good set of html results produced so I might be able to ignore this - or am I missing some important data?

**Comment by wakaleo** [comment]
This is possibly fixed in 1.6.x

**Comment by nbarrett** [comment]
Thanks @wakaleo - this no longer happens in 1.6.4.
